src/auth/providers.js

```diff
@@ -353,7 +375,10 @@ function facebookProvider(auth) {
   };
 }
 
-/** @namespace */
+/**
```

Comment:
[q] Can we somehow get the list of providers in https://adamchel.github.io/stitch-js-sdk/#stitchclient's authenticate method?

Reply:
Just listed them in the docstring